src/components/HomePage/FindSection.tsx

```diff
@@ -1,13 +1,30 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
 import { Input, InputGroup } from 'reactstrap';
+import { getMoviesCreator } from '../../redux/actions/actionCreators/actionCreators';
 import { AppButton, Heading } from '../shared';
 
-export const FindSection: FC = () => (
+export const FindSection: FC = () => {
+  const dispatch = useDispatch();
+
+  const search = useCallback(() => {
+    const input = document.getElementById('input-find-section') as HTMLInputElement;
```

Comment:
useRef instead

Reply:
or state with a value:

const [input, setInput] = useState(''); // '' is the initial state value
...
    <input value={input} onInput={e => setInput(e.target.value)}/>

Reply:
added useRef in commit e9860952613603eb2cf3095f28c180d207b9668c